plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java

```diff
@@ -50,7 +50,9 @@ protected QueryRunner createQueryRunner()
                 ImmutableMap.of(),
                 // This test can only run on a single node, otherwise each node exports its own stats beans and they override each other
                 // You can only bind one such bean per JVM, so this causes problems with statistics being 0 despite backpressure handling
-                1);
+                1,
+                // Use a unique catalog name to prevent other tests from overriding stats bean
```

Comment:
Maybe is clearer?

```suggestion
                // Use a unique catalog name to make sure JMX stats beans are unique and not affected by other tests
```

Reply:
Thanks @hashhar, this is clearer indeed.